libs/ui/components/inputs/button-icon/index.stories.tsx

```diff
@@ -19,7 +19,9 @@ const props: IButtonIconProps = {
     hoverBackgroundColor: '#4C36B5',
     border: 'none',
     borderRadius: '8px',
-    icon: <Icon name='add' stroke='white' width='16' height='16' />,
+    displayIcon: (isHovered) => (
```

Comment:
```suggestion
    displayIcon: () => (
```

don't take unused params

Reply:
Remove it everywhere

Reply:
todo